test/playwright-helpers.ts

```diff
@@ -0,0 +1,16 @@
+import * as path from 'node:path';
+import * as fs from 'fs';
+
+export function getSymfonyKernelVersionId(): number {
+    const kernelPath = path.join(import.meta.dirname, '../test_apps/e2e-app/vendor/symfony/http-kernel/Kernel.php');
```

Comment:
Would use an ENV_VAR here (as its done for KERNEL_CLASS in a way)

Reply:
I thought about it too but re-using `SYMFONY_REQUIRE` env var would makes things counterproductive IMHO:
- it's not very intuitive to pass this env var while running Playwright, I'm sure everyone including me will forgot it
- if you change your Symfony version through `SYMFONY_REQUIRE=x.y.* composer update`,  you must close Playwright and re-open Playwright with `SYMFONY_REQUIRE=x.y.*` 
- there is no native `version_compare()` function in Node.js, we must re-implement the logic ourselves with https://github.com/npm/node-semver for example

The `VERSION_ID` from `test_apps/e2e-app/vendor/symfony/http-kernel/Kernel.php` is the best source of truth we can get when being inside Playwright context